common-arcana.lic

```diff
@@ -109,22 +109,31 @@ module DRCA
     end
   end
 
-  def prepare?(abbrev, mana, symbiosis = false, command = 'prepare', tattoo_tm = false)
+  def prepare?(abbrev, mana, symbiosis = false, command = 'prepare', tattoo_tm = false, runestone_name = '', runestone_tm = false)
     return false unless abbrev
 
     DRC.bput('prep symb', 'You recall the exact details of the', 'But you\'ve already prepared') if symbiosis
 
-    match = DRC.bput("#{command} #{abbrev} #{mana}", get_data('spells').prep_messages)
+    if runestone_name.nil?
+      match = DRC.bput("#{command} #{abbrev} #{mana}", get_data('spells').prep_messages)
+    else
+      match = DRC.bput("#{command} my #{runestone_name}", get_data('spells').invoke_messages)
+    end
     case match
     when 'Your desire to prepare this offensive spell suddenly slips away'
       pause 1
-      return prepare?(abbrev, mana, symbiosis, command, tattoo_tm)
+      return prepare?(abbrev, mana, symbiosis, command, tattoo_tm, data['runestone_name'], data['runestone_tm'])
```

Comment:
❓ Shouldn't these be `runestone_name` and `runestone_tm` instead of `data[...]`?